samples/CommunityToolkit.Maui.Sample/CommunityToolkit.Maui.Sample.csproj

```diff
@@ -7,6 +7,8 @@
     <OutputType>Exe</OutputType>
     <UseMaui>true</UseMaui>
     <SingleProject>true</SingleProject>
+
+    <ImplicitUsings>enable</ImplicitUsings>
```

Comment:
Why not add this to the Build.Props? That way we don't need to duplicate it on all other `csproj`s